nilearn/plotting/img_plotting.py

```diff
@@ -37,6 +39,43 @@
 # Core, usage-agnostic functions
 
 
+def check_threshold(thr, data):
```

Comment:
You should take care of keeping this function generic. For example, the 1e-6 threshold trick should probably not be part of this function because it looks like we don't need it for connectome plotting.